public/electron.js

```diff
@@ -91,6 +91,10 @@ function createWindow () {
             }
           }},
           { type: 'separator' },
+          { label: 'Open dev tools', click: function() {
+            mainWindow.webContents.openDevTools();
```

Comment:
If the user closes and opens the wallet after this blank screen, will it be ok?

Reply:
Yes, but it will look like it's a bug in our app. I know it's good for debugging bug I think I prefer without the menu

Reply:
I don't think it will feel like a bug, as this is a more advanced operation. Since this is still for our testnet, I don't mind having it. For the mainnet I agree we have to remove it.

Reply:
I am ok with or without but I prefer removing it, I do feel is terrible for the user to see a blank screen.

Reply:
Why wouldn't it happen without the dev tools opened? Is it blocked?

I agree that it is terrible for the user to see a blank screen. Maybe we should prevent the reload (or show a custom page explaining what happened).

Reply:
Yes, it's blocked without the dev tools. It only happens when you refresh while using the console (something that I usually do when I am on the browser). That's why I don't like it.

If we manage to prevent this blank screen from appearing, prevent the reload, or show a friendly message to the user, I am ok with it.

Reply:
I'll remove this feature from this PR as it is unrelated and we can open a separate one if we decide we need dev tools.

Reply:
Ok. Remove it and open a new PR so we discuss the best way to include this menu. It's been useful to debug a few issues.

Reply:
Removed it and opened the issue